wiki/fr/fr-Getting-Started.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: wiki
+title: "Pour commencer"
+lang: "fr"
+permalink: "/wiki/Getting-Started"
+---
+
+# Commencer à utiliser Jamulus
+
+## Vous avez besoin de
+
+1. **Un ordinateur** au moins sous **Windows 10**, **MacOS X** El Capitan (v10.11) ou plus réçent, ou **GNU/Linux**.
+1. **Une connection Internet filaire** (n'utilisez pas la Wi-Fi, cela va causer des problèmes de son). La plupart des connexions standard suffisent[^1].
+1. **Des microphones filaires - pas de haut parleurs** pour éviter de l'écho ou des larsen et pour minimiser le délai[^2].
```

Comment:
Missed this, same error as in other places, it is about « casques ou écouteurs filaires » not microphones.
We talk about monitoring. ;)